app/javascript/packages/document-capture/context/native-camera-a-b-test.tsx

```diff
@@ -0,0 +1,40 @@
+import { createContext } from 'react';
+import type { ReactNode } from 'react';
+import useCounter from '../hooks/use-counter';
+
+interface NativeCameraABTestContextInterface {
```

Comment:
Can we suffix this with `Value` instead of `Interface` for consistency with other context value interfaces?

https://github.com/18F/identity-idp/blob/4b13c4f9360e86b67eb46eeb00ca5be8920af63d/app/javascript/packages/document-capture/context/marketing-site.tsx#L16

https://github.com/18F/identity-idp/blob/4b13c4f9360e86b67eb46eeb00ca5be8920af63d/app/javascript/packages/form-steps/form-steps-context.tsx#L3

https://github.com/18F/identity-idp/blob/4b13c4f9360e86b67eb46eeb00ca5be8920af63d/app/javascript/packages/verify-flow/context/address-verification-method-context.tsx#L28

https://github.com/18F/identity-idp/blob/4b13c4f9360e86b67eb46eeb00ca5be8920af63d/app/javascript/packages/verify-flow/context/flow-context.tsx#L3

Reply:
Done in 77dee4339